content/docs/advanced/related-origins/index.md

```diff
@@ -0,0 +1,218 @@
+## Use Cases
+
+The two use cases for Related Origin Requests (RoR) are deployments which use different country code top-level domains (ccTLD) across the world, and deployments where different branding is used for different services.
+
+To address these use cases, it is recommended to leverage industry-standard federation protocols such as [OpenID Connect](https://openid.net/specs/openid-connect-core-1_0.html). This approach facilitates a centralized login experience, by using a dedicated login page (e.g., login.example.com) that serves as the authentication point for all origins and services.
+
+<strong>RoR is designed to be used when federation is <i>not</i> possible.</strong>
+
+{{< callout context="note" title="Websites vs Apps" icon="outline/note" >}} RoR is a WebAuthn feature for the web. App platforms have existing mechanisms for mapping native apps to one or more web origins: [Digital Asset Links](https://developers.google.com/identity/credential-sharing/set-up) for Android and [Associated Domains](https://developer.apple.com/documentation/xcode/supporting-associated-domains) on Apple platforms. {{< /callout >}}
+
+### Country Code Top Level Domains (ccTLDs) {#cctld}
+
+Many global organizations utilize [country code top level domains (ccTLDs)](https://icannwiki.org/Country_code_top-level_domain#Current_ccTLDs) to cater to their international services. For instance, a shopping website might use `shopping.com` for users in the United States, while also having `shopping.ca` for Canada, `shopping.co.uk` for the United Kingdom, `shopping.ie` for Ireland, and `shopping.sg` for Singapore, among others. However, a passkey created on `shopping.com` can't be used  on `shopping.sg`, and vice versa.
+
+### Alternate Branding
+
+Some organizations offer additional services with different or extended branding and share the same accounts. For instance, a shopping site might have their own credit card or their own travel services, which are accessed via different websites.
+
+## How It Works
+
+Related Origin Requests (RoR) works by allowing a Relying Party (RP) to provide a list of valid origins for a given Relying Party ID (RP ID).
+
+During a WebAuthn ceremony, if the RP ID and origin do not match, the WebAuthn client can query the RP for a list of valid origins. The client processes that origin list and then re-evaluates the binding based on this additional context. If an origin is matched, the client will continue with the request in the context of the RP ID.
+
+Labels are the portion of a domain name to the left of the [effective top level domain](https://developer.mozilla.org/en-US/docs/Glossary/eTLD) (commonly referred to as "eTLD+1"). For instance, `shopping` is the label for `shopping.com`, `shopping.co.uk`, `shopping.co.jp`, `shopping.net`, and `shopping.org`. Labels are used as a way to support the large number of entries required to support [ccTLDs](#cctld), while enabling clients to restrict the number of unique origins to prevent abuse.
```

Comment:
I've not come across the term "label" before, but this reads like a "label" is an "eTLD+1". 
However, isn't an eTLD+1 and eTLD + the previous portion of a URL vs. only the portion to the left of an eTLD? 
If so, a "label" is not the same as an eTLD+1.

From mdn:    

    The related concept eTLD+1 means an eTLD plus the next part of the domain name. 
    Because eTLDs are registrable, all domains with the same eTLD+1 are owned by the same organization.

    For example, all the following are eTLD+1 domains:

    - crookedtimber.org
    - theguardian.com
    - sussex.ac.uk
    - aber.ac.uk
    
Here's an illustrative image from [web.dev](https://web.dev/articles/same-site-same-origin#public-suffix-list-etld):
![image](https://github.com/user-attachments/assets/6d84cca4-d50b-486c-819d-2a84382cb497)

It may be worth linking to [https://publicsuffix.org](https://publicsuffix.org/)
since that seems to be where everyone references.

Reply:
Yeah, good call. the:
> commonly referred to as "eTLD+1"

isn't conveying what I originally intended it to. I'll remove it.